packages/scripts/src/admin-proposals/simulateVoteV2.ts

```diff
@@ -0,0 +1,241 @@
+// Description:
+// - Simulate voting affirmatively on any pending Admin Proposals
+const hre = require("hardhat");
+import { computeVoteHashAncillary, getRandomSignedInt } from "@uma/common";
+import { GovernorV2Ethers, VotingTokenEthers, VotingV2Ethers } from "@uma/contracts-node";
+import { BigNumberish, BytesLike, Signer } from "ethers";
+import { getContractInstance } from "../utils/contracts";
+
+const { ethers } = hre;
+
+require("dotenv").config();
+const assert = require("assert").strict;
+
+const SECONDS_PER_DAY = 86400;
+const YES_VOTE = "1";
+
+const FOUNDATION_WALLET = "0x7a3A1c2De64f20EB5e916F40D11B01C441b2A8Dc";
+
+const increaseEvmTime = async (time: number) => {
```

Comment:
nice, this is useful. we can work on adding this to a common util at some point in the future.